content/wasm-languages/python.md

```diff
@@ -91,3 +185,18 @@ Here are some great resources:
 - An in-browser [Python shell in Wasm](https://github.com/ethanhs/python-wasm) (Not the preferred path)
 - One version of [CPython + Wasm](https://github.com/ethanhs/python-wasm), where they are working on [WASI support](https://github.com/ethanhs/python-wasm/issues/18)
 - [SingleStore's wasi-python project](https://github.com/singlestore-labs/python-wasi) is another approach
+
+## Troubleshooting
+
+If you bump into issues when installing the requirements.txt. For example:
+
+```console
+error: externally-managed-environment
+× This environment is externally managed
+```
+
+Please note, this error occurs because installing a **non-brew-packaged** Python package requires you to either:
```

Comment:
```suggestion
Please note, this error is specific to Homebrew-installed Python installations and occurs because installing a **non-brew-packaged** Python package requires you to either:
```